pkg/helm/actions/install_chart_test.go

```diff
@@ -0,0 +1,33 @@
+package actions
+
+import (
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"io/ioutil"
+	"testing"
+)
+
+func TestInstallChart(t *testing.T) {
+	chart := "../testdata/influxdb-3.0.2.tgz"
+	store := storage.Init(driver.NewMemory())
+	actionConfig := &action.Configuration{
+		Releases:     store,
+		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
+		Capabilities: chartutil.DefaultCapabilities,
+		Log:          func(format string, v ...interface{}) {},
+	}
+	rel, err := InstallChart("test-namespace", "test", chart, nil, actionConfig)
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
+1 for tests!  But can we keep the same style?  So far `console` has just used the stdlib `testing` & table tests.  Granted, we are lacking tests in this repo, however this matches the `console-operator` as well, which we also maintain.  

Example from `console` repo:
- https://github.com/openshift/console/blob/master/pkg/server/certs_test.go
Example from `console-operator` repo:
- https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/route/route_test.go#L116

Reply:
fixed https://github.com/openshift/console/pull/3826/commits/028957ef237401b6560830576595e84263996061